db/migrations/20221031120903_reviews.sql

```diff
@@ -0,0 +1,14 @@
+-- migrate:up
+CREATE TABLE reviews (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    score INT NOT NULL,
```

Comment:
각각의 별점 1, 2, 3, 4, 5를 score로 보고 이 스코어를 주었을 때의 데이터 관리는 `rating`이라는 컬럼으로 관리하면 좋을 것 같습니다!

```suggestion
    rating INT NOT NULL,
```